post_proc.py

```diff
@@ -1,8 +1,26 @@
-# Pra remover os espaços em branco no início, lstrip
-
 in_file = open("diario-anadia-2022-08-29-extraido.txt", "r")
-out_text = in_file.read().lstrip()
-out_file = open("diario-anadia-2022-08-29-proc.txt", "w")
-out_file.write(out_text)
-out_file.close()
-    
+#Mudei o nome da variável out_text para in_text, pois ela não será mais usada como texto de saída.
+in_text = in_file.read().lstrip()
+in_text = in_text.splitlines()
+ama_header_words = in_text[0].split()
+ama_header_id = ama_header_words[0]
+def generate_text():
```

Comment:
Essa função não gera as linhas, apenas junta tudo separando por quebra de linha.

Tem uma outra forma bem simples de fazer isso em python, usando list compreensions

https://pythonacademy.com.br/blog/list-comprehensions-no-python

Reply:
Ainda estou com dúvida: é necessário então nós mesmos inserirmos esse cabeçalho no começo de cada município e remover os que estão entre as páginas?

Reply:
O cabeçalho que foi removido foi o segundo a aparecer, não o primeiro, que vem no início do município. Por isso mantive ele.

Reply:
No caso ele fica em cima do município após a exclusão do preâmbulo, mas como as partes estão separadas em pr, ainda contém o preâmbulo antes do nome do município.

Reply:
Está correto sim. Olhei errado essa parte.